mne/cov.py

```diff
@@ -1597,7 +1624,7 @@ def compute_whitener(noise_cov, info, picks=None, rank=None,
     if picks is None:
         # If this changes, we will need to change _setup_plot_projector, too:
         picks = pick_types(info, meg=True, eeg=True, ref_meg=False,
-                           exclude='bads')
+                           seeg=True, exclude='bads')
```

Comment:
ecog, hbo, hbr?

Hopefully now this can just be `_pick_data_channels` or whatever that private function is called

Reply:
+1